Header header
float32 ActualSpd		# m/s  VCU反馈车体实际速度
bool EStopStatus		# 紧停状态